src/tools/dart-compile.md

```diff
@@ -57,6 +57,10 @@ The `dart compile` command replaces the
   see the [platforms discussion](/overview#platform).
 {{site.alert.end}}
 
+Refer to the [native_app] sample for a simple example of using `dart create`
+to compile and run a native app.
```

Comment:
This implies that `dart compile` runs the app. Maybe:

```suggestion
to compile a native app.
```

We could get fancier/clearer:

```suggestion
to compile a native app, followed by examples of running the app.
```